addons/binding/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/handler/PulseaudioBridgeHandler.java

```diff
@@ -0,0 +1,169 @@
+import org.openhab.binding.pulseaudio.internal.items.AbstractAudioDeviceConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PulseaudioBridgeHandler extends BaseBridgeHandler {
+	private Logger logger = LoggerFactory.getLogger(PulseaudioBridgeHandler.class);
+	
+	public final static Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.singleton(PulseaudioBindingConstants.BRIDGE_THING_TYPE);
+	
+	public String host;
+    public int port;
+    
+    public int refreshInterval;
+    
+    private PulseaudioClient client;
+    
+    private HashSet<String> lastActiveDevices = new HashSet<String>();
+    
+    private ScheduledFuture<?> pollingJob;
+	private Runnable pollingRunnable = new Runnable() {
+		@Override
+		public void run() {
+			client.update();
+			for (AbstractAudioDeviceConfig device : client.getItems()) {
+				if (lastActiveDevices!=null && lastActiveDevices.contains(device.getPaName())) {
+					for (DeviceStatusListener deviceStatusListener : deviceStatusListeners) {
+						try {
+							deviceStatusListener.onDeviceStateChanged(getThing().getUID(), device);
+						} catch (Exception e) {
+							logger.error("An exception occurred while calling the DeviceStatusListener", e);
+						}
+					}
+				}
+				else {
+					for (DeviceStatusListener deviceStatusListener : deviceStatusListeners) {
+						try {
+							deviceStatusListener.onDeviceAdded(getThing(), device);
+							deviceStatusListener.onDeviceStateChanged(getThing().getUID(), device);
+						} catch (Exception e) {
+							logger.error("An exception occurred while calling the DeviceStatusListener", e);
+						}
+						lastActiveDevices.add(device.getPaName());
+					}
+				}
+			}
+		}
+	};
+	
+	private List<DeviceStatusListener> deviceStatusListeners = new CopyOnWriteArrayList<>();
+
+	public PulseaudioBridgeHandler(Bridge bridge) {
+		super(bridge);
+	}
+
+	@Override
+	public void handleCommand(ChannelUID channelUID, Command command) {
+		if (command instanceof RefreshType) {
+			logger.debug("Refresh command received.");
+			client.update();
+			// TODO: postUpdate von neuem State an Item
+			
+		} else
+			logger.warn("No bridge commands defined.");
+	}
+	private synchronized void startAutomaticRefresh() {
+		if (pollingJob == null || pollingJob.isCancelled()) {
+			pollingJob = scheduler.scheduleAtFixedRate(pollingRunnable, 0, refreshInterval, TimeUnit.MILLISECONDS);
+		}
+	}
+	
+	public AbstractAudioDeviceConfig getDevice(String name) {
+		return client.getGenericAudioItem(name);
+	}
+	
+	public PulseaudioClient getClient() {
+		return client;
+	}
+	
+	@Override
+	public void initialize() {
+        logger.debug("Initializing Pulseaudio handler.");
+		Configuration conf = this.getConfig();
+		super.initialize();
+		
+		this.host = String.valueOf(conf.get(BRIDGE_PARAMETER_HOST));
+		
+		try {
+			this.port = Integer.parseInt(String.valueOf(conf.get(BRIDGE_PARAMETER_PORT)));
```

Comment:
conf.get should actually return you BigDecimals, so no need to parse this as a String